requirements.txt

```diff
@@ -143,8 +143,8 @@ zope.event==4.5.0
 zope.interface==5.4.0
 
 # Build dependencies
-Cython==3.0a9
-flit_core==3.7.1
+Cython==3.0.0a9
+flit-core==3.7.1
```

Comment:
Is this right? I'd confirmed that `pip install flit_core==3.7.1` installs. https://pypi.org/project/flit_core/